### DISCUSSION TITLE
filetype_to_parsername is deprecated, please use 'vim.treesitter.language.register'

### DISCUSSION THREAD

**Discussion by hrcHarry** [ranjithshegde/ccls.nvim#12] [Q&A]
Hi
When I use `:CclsIncomingCallsHierarchy float`, it gives the floating window showing the incoming calls, but neovim also gives the nvim-treesitter info: `filetype_to_parsername is deprecated, please use 'vim.treesitter.language.register'`.

How to fix it? Thanks.

**Comment by ranjithshegde** [comment]
If you are using nvim 0.9 or newer, the latest commit should fix this issue

**Reply by hrcHarry** [reply]
Fixed. Thanks! 😀
